client/role.go

```diff
@@ -0,0 +1,76 @@
+package client
+
+type RoleCreatePayload struct {
+	Name           string   `json:"name"`
+	OrganizationId string   `json:"organizationId"`
+	Permissions    []string `json:"permissions"`
+	IsDefaultRole  bool     `json:"isDefaultRole"`
+}
+
+type RoleUpdatePayload struct {
```

Comment:
you can use this as a base to other types

Reply:
@GiliFaroEnv0 - it's not as clean in golang to do.
I'll see if I can improve it a little in the next PR. (Will merge this and review in the next PR).